apps/src/sites/studio/pages/levels/_free_response.js

```diff
@@ -0,0 +1,85 @@
+import $ from 'jquery';
+import Parser from '@code-dot-org/redactable-markdown';
+import defaultSanitizationSchema from 'hast-util-sanitize/lib/github.json';
+import firehoseClient from '@cdo/apps/lib/util/firehose';
+import rehypeParse from 'rehype-parse';
+import rehypeRaw from 'rehype-raw';
+import rehypeSanitize from 'rehype-sanitize';
+import rehypeStringify from 'rehype-stringify';
+import remarkRehype from 'remark-rehype';
+
+// In preparation for switching FreeResponse levels over to use the
+// SafeMarkdown renderer, compare clientside-rendered markdown against
+// serverside-rendered markdown. Disparities will be logged to firehose so
+// markdown can be updated to work on both renderers, and the eventual
+// switchover will be guaranteed to be seamless.
```

Comment:
creative approach! how long do you plan to wait, to ensure that all free response levels are visited? would it improve coverage (or decrease wait time) at all to put this branch up on an adhoc, and hit each free response level url there instead of waiting for it to happen naturally in production?

Reply:
I'm definitely planning on hitting all levels manually at some point before resolving this just to double-check; the primary advantage of doing it this way rather than relying solely on manual checking is that we'll be able to catch any regressions that happen in between the manual test and the resolution.

I'm thinking I'll leave it running until we've gone a day or two without any new notifications, fixing up levels as they pop up, and then do a thorough pass on all levels for verification.